pkg/recommender.py

```diff
@@ -425,3 +436,68 @@ def record_data(self, data):
 
     storing_db.close()
 
+  def _schedule_evt(self):
+    '''
+    Send the morning message at 10 am
+    '''
+
+    while True:
+      t = datetime.now()
+      if (t.hour == 10 and t.minute == 2): #CHANGE TO SPECIFIC HOUR WE WANT TO SEND MESSAGE
```

Comment:
this logic may not be perfect. Say if I start the system on `8 am` in the morning, this condition will be False and the program will skip to the `sleep` which sleep 24 hours, another `8 am`....

in my pseudo code below, we should check how long should we sleep to the next `10 am`, which is tricky, you may or may not need to skip day, say start at `8 am` vs `11 am`. No more further condition checking after awake.
```python
def _schedule_evt():
  while True:
    now = datetime.now()
    next_time = now.next_morning()
    time.sleep(next_time - now)

    send_recommendation()
```